.env.test

```diff
@@ -10,6 +10,10 @@ DB_HOST=db
 REDIS_HOST=redis
 REDIS_PORT=6379
 REDIS_DB=0
+STATSD_BACKEND=markus.backends.datadog.DatadogMetrics
```

Comment:
Mostly using this Python dotted path as a placeholder for what we need to add to the env for production.